doc/development/building.rst

```diff
@@ -0,0 +1,16 @@
+Building Applications
+=====================
+
+`Jenkins <https://tssw-ci.lsst.org/>`_ is the chosen Continuous Integration platform.
+Each application should have a build project in Jenkins.
```

Comment:
I think you could turn this into a bullet point

Reply:
Done